### ISSUE TITLE
suprising md3 loading behavior (trying to load <basename>_[123].md3)

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#52]
All these models are there, and it looks like they are correctly displayed in-game (hence correctly loaded).

```
Debug: Failed to open 'models/fx/metal_gibs/m_gib8_3.md3' for reading: No such file or directory 
Debug: Failed to open 'models/fx/metal_gibs/m_gib8_2.md3' for reading: No such file or directory 
Debug: Failed to open 'models/fx/metal_gibs/m_gib8_1.md3' for reading: No such file or directory 
Debug: Failed to open 'models/fx/alien_gibs/a_gib4_3.md3' for reading: No such file or directory 
Debug: Failed to open 'models/fx/alien_gibs/a_gib4_2.md3' for reading: No such file or directory 
Debug: Failed to open 'models/fx/alien_gibs/a_gib4_1.md3' for reading: No such file or directory 
```

Any md3 laying in package raises this error:

```
Debug: Failed to open 'models/generic/sphere_3.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphere_2.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphere_1.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphericalCone64_3.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphericalCone64_2.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphericalCone64_1.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphericalCone240_3.md3' for reading: No such file or directory 
Debug: Failed to open 'models/generic/sphericalCone240_2.md3' for reading: No such file or directory 
```

It does not prevent the game to be played though.

The error is displayed while using `set logs.logLevel.default debug`.

**Comment by illwieckz** [comment]
hmmm, no, false report, they use slightly renamed names…

**Comment by illwieckz** [comment]
Oh no, it's a true issue, but I don't know why. For every `filename.md3` that exists, the engine looks for `filename_[123].md3`, why?

**Comment by DolceTriade** [comment]
It's a quake 3 relic I believe. This is intentional. Tries to load different LoDs: https://github.com/DaemonEngine/Daemon/blob/master/src/engine/renderer/tr_model.cpp#L174

**Comment by illwieckz** [comment]
Oh I see, the message can probably be more helpful (like having some words telling it's only looking for optional models).

Do we need to keep that relic?

**Comment by illwieckz** [comment]
Well, even unvanquished does not need that, that's probably a bad idea to remove features from the engine itself. Would be cool to have a clearer message though, I don't know if it's possible to pass an option extra argument to `FS_ReadFile()` to tell it this file is optional (so it can adjust the words used).

**Comment by DolceTriade** [comment]
FS_ReadFile is meant to be called to fail. Verbose/Debug logs are such for a reason.

**Comment by illwieckz** [comment]
fixed by #72

**Comment by Viech** [comment]
There will only be peace when `grep -Ri md3` returns with exit code `1`.
